src/Elasticsearch/Utility.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Elasticsearch PHP client
+ *
+ * @link      https://github.com/elastic/elasticsearch-php/
+ * @copyright Copyright (c) Elasticsearch B.V (https://www.elastic.co)
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ * @license   https://www.gnu.org/licenses/lgpl-2.1.html GNU Lesser General Public License, Version 2.1 
+ * 
+ * Licensed to Elasticsearch B.V under one or more agreements.
+ * Elasticsearch B.V licenses this file to you under the Apache 2.0 License or
+ * the GNU Lesser General Public License, Version 2.1, at your option.
+ * See the LICENSE file in the project root for more information.
+ */
+declare(strict_types = 1);
+
+namespace Elasticsearch;
+
+class Utility
+{
+    const ENV_URL_PLUS_AS_SPACE = 'ES_URL_PLUS_AS_SPACE';
```

Comment:
We've used `ELASTIC_CLIENT_` as our prefix for env vars historically, what are your thoughts on using that prefix here ad well?

Reply:
Sounds good!